src/redux/books/AddBookBtn.js

```diff
@@ -0,0 +1,16 @@
+import { useDispatch } from 'react-redux';
+import { addBook } from 'redux/books/bookSlice';
+
+const AddBookBtn = (id, title, author) => {
+  const dispatch = useDispatch();
+  const handleAddBook = () => dispatch(addBook(id, title, author));
+  return (
+    <button
+      onClick={handleAddBook}
+      type="button"
+    >
+      Add Book
+    </button>
+  );
+};
```

Comment:
- [ ] Please makes sure this component is in the `src/components` directory since it is the directory dedicated to contain all your app components :pray: